guide/docs/popular-topics/intents.mdx

```diff
@@ -187,9 +192,29 @@ client = disnake.Client(intents=intents)
 # or
 from disnake.ext import commands
 
-bot = commands.Bot(command_prefix="!", intents=intents)
+bot = commands.Bot(command_prefix=commands.when_mentioned, intents=intents)
 ```
 
+### Why do most messages have no content?
+
+As of April 30th 2022, Discord has blocked message content from being sent to bots that do not declare the :attr:`Intents.message_content` intent when connecting to discord.
```

Comment:
Simply put, the Sphinx thingy wouldn't apply here.

```suggestion
As of April 30th 2022, Discord has blocked message content from being sent to bots that do not declare the <DocsLink reference="disnake.Intents.message_content">message_content</DocsLink> intent when connecting to discord.
```

Reply:
https://github.com/DisnakeDev/guide/pull/30/commits/4911d5fe2ed73f6fc0c12ef596a55c7ff93fc2b8